packages/fuel-indexer-macros/src/indexer.rs

```diff
@@ -696,217 +702,231 @@ fn process_fn_items(
-                            fuel::Receipt::Call { id: contract_id, amount, asset_id, gas, param1, to: id, .. } => {
-                                #check_if_subscribed_to_contract
-
-                                let fn_name = decoder.selector_to_fn_name(param1);
-                                return_types.push(param1);
-                                callees.insert(id);
-
-                                let data = serialize(
-                                    &Call {
-                                        contract_id: ContractId::from(<[u8; 32]>::from(contract_id)),
-                                        to: ContractId::from(<[u8; 32]>::from(id)),
-                                        amount,
-                                        asset_id: AssetId::from(<[u8; 32]>::from(asset_id)),
-                                        gas,
-                                        fn_name
-                                    }
-                                );
-                                let ty_id = Call::type_id();
-                                decoder.decode_type(ty_id, data);
-                            }
-                            fuel::Receipt::Log { id, ra, rb, .. } => {
-                                #check_if_subscribed_to_contract
-                                let ty_id = Log::type_id();
-                                let data = serialize(
-                                    &Log {
-                                        contract_id: ContractId::from(<[u8; 32]>::from(id)),
-                                        ra,
-                                        rb
-                                    }
-                                );
-                                decoder.decode_type(ty_id, data);
-                            }
-                            fuel::Receipt::LogData { rb, data, ptr, len, id, .. } => {
-                                #check_if_subscribed_to_contract
-                                decoder.decode_logdata(rb as usize, data.unwrap_or(Vec::<u8>::new()));
-                            }
-                            fuel::Receipt::Return { id, val, pc, is } => {
-                                #check_if_subscribed_to_contract
-                                if callees.contains(&id) {
-                                    let ty_id = Return::type_id();
-                                    let data = serialize(
-                                        &Return {
-                                            contract_id: ContractId::from(<[u8; 32]>::from(id)),
-                                            val,
-                                            pc,
-                                            is
-                                        }
-                                    );
-                                    decoder.decode_type(ty_id, data);
+            use anyhow::Context;
+            use fuel_indexer_utils::plugin::types::fuel::TransactionData;
```

Comment:
All fuel/explorer types are available via the plugin under the `fuel` namespace
```suggestion
            use fuel::TransactionData;
```